lib/src/host_reachability_checker.dart

```diff
@@ -32,19 +33,49 @@ final _defaultUrls = List<ConnectionEntry>.unmodifiable([
 ]);
 
 abstract class HostReachabilityChecker {
```

Comment:
Since the method is shown to the outside world and is public, I think you can add some documentation above it - just like with everything else from `connection_checker.dart` which is public and shown by us outside